apps/ledger-live-mobile/src/components/RootNavigator/AccountsNavigator.tsx

```diff
@@ -75,6 +76,13 @@ export default function AccountsNavigator() {
           headerShown: false,
         }}
       />
+      <Stack.Screen
```

Comment:
make it render based on the feature flag please

Reply:
Done